test/run_tests.sh

```diff
@@ -168,6 +168,7 @@ function run_op_tests {
   run_test "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
   run_test_without_functionalization "$CDIR/test_operations.py" "$@" --verbosity=$VERBOSITY
   run_test "$CDIR/test_async_closures.py"
+  run_test "$CDIR/test_autocast.py"
```

Comment:
Thanks, did you run this test on TPUVM to verified it passed as expected? I copied most of the test case from a gpu test, hence I am not sure if those test cases are also suitable for the TPU.

Reply:
The TPU tests follow the same format as the GPU tests, but I modified the list of ops that are tested for TPUs. I did verify the TPU tests pass on the TPUVM.